clusterman/kubernetes/kubernetes_cluster_connector.py

```diff
@@ -498,6 +499,20 @@ def _is_unschedulable(self, pod: KubernetesPod) -> bool:
                 return True
         return False
 
+    def get_node_priority(self, node_id: str) -> float:
+        pods = self._pods_by_ip[node_id]
+        if not pods or len(pods) == 0:
+            return 0.0
+        max_requested_resource = max([self.get_pod_resources_score(pod) for pod in pods])
```

Comment:
why use only the single biggest pod to gauge priority why not e.g. calculate the priority as a product of how many pods (more disruption) and their average size?

Reply:
I was thinking about average/max/Percentile XX and etc. Actually none of them can solve problem totally. I just wanted to do an experiment with it. 
I'm currently doing some science: 
* Marking node killable only if its existing pods can be scheduled to other nodes.
Which is really not easy with our current architecture